datavalidation.go

```diff
@@ -256,30 +281,33 @@ func (f *File) AddDataValidation(sheet string, dv *DataValidation) error {
 	if err != nil {
 		return err
 	}
+	f.mu.Lock()
+	defer f.mu.Unlock()
```

Comment:
All concurrency safety functions have a note in the documentation. The AddDataValidation function is not concurrency safety currently, so these changes should be a feature but not a bug, and I certainly accept this feature proposal. I suggest adding a note in the documentation for this function:

```go
// AddDataValidation provides set data validation on a range of the worksheet
// by given data validation object and worksheet name. This function is 
// concurrency safe. The data validation object can be created by
// NewDataValidation function.
```

Reply:
Thanks for your suggestion. I have added in the new  commit.